### ISSUE TITLE
[css-break] Rename and/or reimagine `widows` and `orphans`

### ISSUE THREAD

**Issue by kizu** [w3c/csswg-drafts#11286]
I am splitting this from the issue about renaming `avoid-orphans`: https://github.com/w3c/csswg-drafts/issues/11283

For context and arguments why we want to rename it, see the linked issue, but what I would like to discuss in this one is twofold:

1. Do we want to rename the `widows` and `orphans` properties from the [CSS Fragmentation Module](https://drafts.csswg.org/css-break/#widows-orphans)?

    The exact names need to be bikeshed, but because both of these are about “bad breaks” (how Ellen Lupton names them in her “Thinking with Type”), we could name it by it. Or, we could go from the current behavior which “keeps the lines together”, and name it something like `min-trailing-lines` and `min-leading-lines`. Or something else!

2. If so, do we want to create new properties with the same syntax, and deprecate the old ones, keeping them as aliases? Or is there something else we could improve with the properties, given we will not be bound by interoperability?

Given, the current support for these properties is [very spotty](https://caniuse.com/css-widows-orphans), either could work out. I do not know the reason Firefox did not implement it yet, as well as why there are differences in what was implemented in Chrome and Safari (see https://quirksmode.org/css/css2/widows.html for test cases).

This could be a good opportunity to discuss these differences, why they're there, and if we could come with a consensus which all browsers could agree on, so we could move forward.

Both avoiding short lines, and avoiding single leading or trailing lines in fragments are something I (and many other developers) wanted to have for a while (example: https://github.com/web-platform-tests/interop/issues/520), so it would be great to get closer to it.

**Comment by Crissov** [comment]
Note that Can I Use does not track HTML+CSS to PDF generators. I think support there is way better than ”spotty“.

**Comment by AmeliaBR** [comment]
Yes, given the wide use of these properties in print formatters, compatibility is relevant. The easiest approach would be to define synonym properties with matching syntax: two separate properties, each specifying a number of lines in a paragraph before or after a column or page break in order for that to be an acceptable break point. The old names would continue to be supported (or added as synonyms when support is implemented), but deprecated.

I do agree that the current names are kind of awful for anyone who hasn't studied typography in an English language context. And maybe even for those who have.

I like the idea of `min-lines` as a basis for the names. However, I worry that trailing / leading is still confusing, when we are talking about the trailing lines of a paragraph leading the start of a column or page and the leading lines of a paragraph trailing on the end of a column or page.

Proposal (slightly verbose but clear): `min-lines-before-break` and `min-lines-after-break`

**Comment by bfgeek** [comment]
Practically speaking as people target specific browsers/renderers for print/pdf-generation use-cases (who do support these properties) renaming these isn't viable.

**Comment by frivoal** [comment]
I think I like the suggestion from https://github.com/w3c/csswg-drafts/issues/11286#issuecomment-2505011440 best. In any case, we'd have to keep the old bad names as [aliases](https://www.w3.org/TR/css-cascade-4/#aliasing).

**Comment by zcorpan** [comment]
After reading the spec for `orphans` and `widows`, without reading the comments above, I came up with these names:

- `lines-before-break`
- `lines-after-break`

I considered a `min-` prefix but I think `min-` implies that there is also a `max-` (and no prefix), which doesn't seem ideal. `minimum-` could work but makes the names quite long and the names above are probably clear enough.

After reading the comments I see @AmeliaBR proposed the same names (with `min-` prefix).
